### PR TITLE
Remove functionality now duplciated with pulumi/aws.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Going to do this a type at a time, so i can see if i run into any issues as i go along.

File: Gopkg.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [[constraint]]
   name = "github.com/pulumi/pulumi"
-  version = "v0.15.1"
+  revision = "e498cab2390aa24546fbc3d7950c4800b0ba3e5d"
```
Comment (CyrusNajmabadi) [inline Gopkg.toml]: @ellismg Is there a more appropriate way to do this?  I need the latest pulumi/pulumi bits to fix a test harness issue.

**Comment by CyrusNajmabadi** [comment]
@lukehoban can you take a look?  Thanks!

File: aws/topic.ts [hunk 2]
```diff
@@ -42,15 +41,21 @@ export class Topic<T> extends pulumi.ComponentResource implements cloud.Topic<T>
                 TopicArn: topicId.get(),
             }).promise();
         };
+
+        this.registerOutputs({
+            topic: this.topic,
```
Comment (swgillespie) [inline aws/topic.ts]: should we register `subscriptions` too?

File: Gopkg.toml (hunk 1 above)
Comment (CyrusNajmabadi) [reply Gopkg.toml]: Done!

**Pull request merged**